### PR TITLE
04/add new course

### PR DESCRIPTION
**Pull request by rorysproule1**

### CODE REVIEW

File: addCourse.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<html>
+<form>
+    <fieldset>
+        <legend>Add Course</legend>
+        Title: <input type="text" name="Title" size="50" maxlength="101" required><br>
```
Comment (ghost) [inline addCourse.html:18]: The acceptance criteria state that max title length is 100.

**Review by jurassic-marc — COMMENTED** [review]
Seems fine

File: addCourse.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        Date: <input type="date" name="Date" size="11" maxlength="11" required><br>
+
+        Description: <input type name="Description" size="50" maxlength="300" required><br>
+        NameOfTrainer: <input type="text" name="NameOfTrainer" size="50" maxlength="100" required><br>
```
Comment (ghost) [inline addCourse.html:22]: The customer changed the acceptance criteria very recently so name of trainer should no longer be present in the frontend.

File: addCourse.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import "db.js"
+function callSqlInsert(data)
+{
+    alert("called");
+    //assuming data is done
+    var dummyData = ['Course title', '30/07/19','Belfast','This is a course test', 'Owner','Software Engineers',10];
+    var tableName = "Course";
+    var queryName = "INSERT INTO " + tableName + "(Title, Date, Location, Description, Owner, TargetAudience) \nVALUES(";
```
Comment (ghost) [inline addCourse.js]: This doesn't appear to be a complete query. What's the purpose of this?

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request merged**